pages/_document.tsx

```diff
@@ -10,9 +10,23 @@ import createEmotionCache from '../utils/createEmotionCache.ts';
 
 export default class MyDocument extends Document {
   render() {
+    const GTM_ID = process.env.NEXT_PUBLIC_GTM_ID;
+
     return (
       <Html lang="en-US">
         <Head>
+           {/* Google Analytics */}
```

Comment:
I would also try saving to see if ESLint auto-corrects this line, it looks like there is an issue with the indentation.

Reply:
Ok, will do